tests/sdk/test_client.py

```
@@ -0,0 +1,182 @@
+from sdks.python.src.honcho.async_client.client import AsyncHoncho
+from sdks.python.src.honcho.async_client.pagination import AsyncPage
+from sdks.python.src.honcho.async_client.peer import AsyncPeer
+from sdks.python.src.honcho.async_client.session import AsyncSession
+from sdks.python.src.honcho.client import Honcho
+from sdks.python.src.honcho.pagination import SyncPage
+from sdks.python.src.honcho.peer import Peer
+from sdks.python.src.honcho.session import Session
+
+
+@pytest.mark.asyncio
+async def test_client_init(client_fixture, client: TestClient):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_client_init`

Add return type annotation: `None`

(ANN201)